test/local_integration_test.py

```diff
@@ -44,32 +65,36 @@ def setUp(self):
         super().setUp()
         self.set_lambda_test_mode(True)
         self.bundle_uuid_prefix = ''.join([str(random.choice('abcdef0123456789')) for _ in range(self.prefix_length)])
+        self.indexed_uuids_to_version = {}
```

Comment:
I have the feeling that returning fqids from the reindex method will allow this to be a set instead.

Reply:
This is true.